lib/Parser/rterrors.h

```diff
@@ -20,6 +20,7 @@
 #define RT_ERROR_MSG_UNUSED_ENTRY
 #endif
 
+// NOTE: The VBSERR macros should not be changed due to legacy dependencies outside of Chakra.
```

Comment:
Perhaps we should explain the naming was for legacy reasons then. I think that's the primary surprise for most people.